### PR TITLE
[Add-ins] Add section on verifying domain

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
Addresses issue https://github.com/OfficeDev/outlook-dev-docs/issues/252

### CODE REVIEW

**Comment by jasonjoh** [comment]
OPS Build status updates of commit _[dc1beb3](https://github.com/OfficeDev/outlook-dev-docs/pull/513/commits/dc1beb3c208b12bb346dd8a0d711de57b6268aa3)_: 

:clock10: **Preparing**: average preparing time is _41 sec(s)_

**Comment by jasonjoh** [comment]
OPS Build status updates of commit _[dc1beb3](https://github.com/OfficeDev/outlook-dev-docs/pull/513/commits/dc1beb3c208b12bb346dd8a0d711de57b6268aa3)_: 

:clock10: **Incremental building**: average incremental building time is _18 sec(s)_

**Comment by jasonjoh** [comment]
OPS Build status updates of commit _[dc1beb3](https://github.com/OfficeDev/outlook-dev-docs/pull/513/commits/dc1beb3c208b12bb346dd8a0d711de57b6268aa3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/validate-an-identity-token.md](https://github.com/OfficeDev/outlook-dev-docs/blob/elizs-verify-domain/docs/add-ins/validate-an-identity-token.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C1%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201902012301173300-513%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=GQbb5MnHoKXcjMhN3SECE05i%2F%2FY%2FjFyxtMcaH9dTzXI%3D&st=2019-02-01T22%3A57%3A18Z&se=2019-03-04T23%3A02%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/add-ins/validate-an-identity-token.md [hunk 1]
```diff
@@ -44,6 +44,14 @@ To validate the token contents, you should check the following.
     - Check that the `version` claim inside the `appctx` claim is set to `ExIdTok.V1`.
     - Check that the `amurl` claim inside the `appctx` is set to a valid URL.
 
+### Verify the domain
+
+You could also consider verifying that the domain of the `amurl` claim matches the user's domain by getting the list of endpoints then trying them in a specific order until you find the one that works.
+
```
[the remark this answers is not in the document]
Comment (ElizabethSamuel-MSFT) [reply docs/add-ins/validate-an-identity-token.md]: @kbrandl: However, I didn't think Autodiscover applies to O365 which is why it's not mentioned until I discuss the Exchange option...

[the remark this answers is not in the document]
Comment (kbrandl) [reply docs/add-ins/validate-an-identity-token.md]: [This article in the docs](https://docs.microsoft.com/en-us/exchange/client-developer/exchange-web-services/autodiscover-for-exchange#options-for-using-autodiscover) suggests that Autodiscover can indeed apply to O365 ("Exchange Online as part of Office 365" is mentioned in the **Pros** column of Table 3).

**Comment by jasonjoh** [comment]
OPS Build status updates of commit _[abf5569](https://github.com/OfficeDev/outlook-dev-docs/pull/513/commits/abf556923bc61d195eb1cfbc6ba2793edec8e9b5)_: 

:clock10: **Preparing**: average preparing time is _41 sec(s)_

**Comment by jasonjoh** [comment]
OPS Build status updates of commit _[abf5569](https://github.com/OfficeDev/outlook-dev-docs/pull/513/commits/abf556923bc61d195eb1cfbc6ba2793edec8e9b5)_: 

:clock10: **Incremental building**: average incremental building time is _18 sec(s)_

**Comment by jasonjoh** [comment]
OPS Build status updates of commit _[abf5569](https://github.com/OfficeDev/outlook-dev-docs/pull/513/commits/abf556923bc61d195eb1cfbc6ba2793edec8e9b5)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/validate-an-identity-token.md](https://github.com/OfficeDev/outlook-dev-docs/blob/elizs-verify-domain/docs/add-ins/validate-an-identity-token.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C5%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201902051741146605-513%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=5MT7UnpcWNA5ZRdBNqaF%2FcWWY%2B%2F48Eg4lNd0xm%2FtKdI%3D&st=2019-02-05T17%3A37%3A18Z&se=2019-03-08T17%3A42%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**